src/domains/transaction/hooks/use-profile-transactions.ts

```diff
@@ -177,11 +184,18 @@ export const useProfileTransactions = ({ profile, wallets, limit = 30 }: Profile
 			};
 
 			if (transactionType && transactionType !== "all") {
-				queryParameters.type = transactionType;
+				query.type = transactionType;
 			}
 
+			if (queryParameters?.recipientId) {
+				// @ts-ignore
+				query.recipientId = queryParameters.recipientId;
```

Comment:
@ItsANameToo ignoring ts error here because although SDK accepts `recipientId` as a parameter, it's not exposed in type.